mix.exs

```diff
@@ -38,7 +38,8 @@ defmodule ConfigCat.MixProject do
 
   def application do
     [
-      extra_applications: [:logger]
+      extra_applications: [:logger],
+      mod: {ConfigCat.Application, []}
```

Comment:
Now that we have an `Application` module that needs to start automatically, we need to add this config setting so that happens.